ui/DesignSystem/Component/BigHeader.svelte

```diff
@@ -0,0 +1,96 @@
+<script>
+  export let style = null;
+  export let data = null;
+  export let variant = null; // profile | org
+
+  import { Avatar, Icon, Title, Text, Numeric } from "../Primitive";
+</script>
+
+<style>
+  .header {
+    display: flex;
+    flex-direction: column;
+    width: calc(100vw - var(--sidebar-width));
+    height: 261px;
```

Comment:
Where's this number from? Kind of a strange number :thinking:

Reply:
resolved in [80abf4a](https://github.com/radicle-dev/radicle-upstream/pull/494/commits/80abf4a62ab6607f079bd6c6c8d840b554742fae)